packages/kbn-unified-data-table/src/components/json_code_editor/json_code_editor_common.tsx

```diff
@@ -12,12 +12,11 @@ import React from 'react';
 import { i18n } from '@kbn/i18n';
 import { monaco, XJsonLang } from '@kbn/monaco';
 import { EuiButtonEmpty, EuiCopy, EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
-import { CodeEditor } from '@kbn/kibana-react-plugin/public';
-
-const codeEditorAriaLabel = i18n.translate('discover.json.codeEditorAriaLabel', {
+import { CodeEditor } from '@kbn/code-editor';
```

Comment:
FYI @lukasolson , you know we recently chatted about refactoring to `@kbn/code-editor` , @YulNaumenko did it, and the tests pass ... this is great, so one less blocker to move more stuff to the UnifiedDocViewer package!

Reply:
@YulNaumenko I think this change introduced a regression. When expanding a document, the JSON part is no longer rendered with full height:
<img width="635" alt="Bildschirmfoto 2023-08-28 um 16 23 40" src="https://github.com/elastic/kibana/assets/463851/ad90f795-9c44-4dc6-96bf-edca7d928a99">

Reply:
Resolved with the commit https://github.com/elastic/kibana/pull/163211/commits/6a0abe07e648f7f3112e8747c69a09eb31579952